templates/init-registry-org.py.j2

```diff
@@ -0,0 +1,60 @@
+#!/usr/bin/env python
+
+from argparse import ArgumentParser, RawTextHelpFormatter
+
+import json
+import logging
+import requests
+
+HELP_DEFINITION='''
+Script to initiate Gitea Image registry organization
+'''
+HELP_EXAMPLE='''
+Example:
+\n\t./init-regisitry-org.py 
+  --org org1,org2,org3 \
+  --url https://example.com/gitea \
+  --user username --password password
+'''
+
+headers= {
+    'Content-type': 'application/json'
+}
+
+data={
+    'visibility': 'limited'
+}
+
+def create_org(url, org, basic):
+    org_data=data
+    org_data['username'] = org
+    logging.info('Creating organisation %s' % org)
+    logging.debug('Argument:%s' % json.dumps(data))
+    res=requests.post('%s/api/v1/orgs' % url, auth=basic,headers=headers,data=json.dumps(org_data))
+    res.raise_for_status()
+    return res
+
+
+def parse_args():
+    parser = ArgumentParser(description=HELP_DEFINITION, epilog=HELP_EXAMPLE, formatter_class=RawTextHelpFormatter)
+    parser.add_argument('-o', '--org',      required=True,  help='List of organization to create in Gitea. Separate each value with ","')
+    parser.add_argument('-U', '--url',      required=True,  help='URL of the Gitea instance')
+    parser.add_argument('-u', '--user',     required=True,  help='Username')
+    parser.add_argument('-p', '--password', required=True,  help='Password')
+    parser.add_argument('-l', '--log',      required=False, default='INFO')
+    return parser.parse_args()
+
+
+def main():
+    args = parse_args()
+    logging.basicConfig(level=args.log.upper())
+    basic = requests.auth.HTTPBasicAuth(args.user, args.password)
+    orgs=args.org.split(',')
+    for org in orgs:
+        try:
+            create_org(args.url, org, basic)
+        except requests.exceptions.HTTPError as err:
+            logging.error('Error %s when trying to create the org %s: %s' % (err.response.status_code, org, err.response.text))
```

Comment:
Doesn't this mean that the failure is silent because despite errors the script will return 0?

We should probaby re-raise the exception, or just use `sys.exit(1)`.

Reply:
The API return a code 422 when the organization already exist, that's why it fail silently. But i agree, it's not great, gonna update this

Reply:
It's better tho handle the 422 code separately in the `except` clause then.